src/internal_modules/roc_core/target_nocsprng/roc_core/secure_random.cpp

```diff
@@ -81,5 +80,20 @@ double fast_random_gaussian() {
     return r * std::cos(theta);
 }
 
+// Wrappers
+// To use for external calls
+
+uint32_t secure_random() {
+    return fast_random();
+};
+
+uint32_t secure_random_range(uint32_t from, uint32_t to) {
+    return fast_random_range(from, to);
+};
+
+double secure_random_gaussian() {
+    return fast_random_gaussian();
+};
+
```

Comment:
this should be implemented in target_nocsprng (as written in #683)

Reply:
Hello, i don't understand. That is target_nocsprng.